gulp-tasks/build.js

```diff
@@ -1,37 +1,55 @@
-var gulp = require('gulp'),
-    minCSS = require('gulp-clean-css'),
-    jshint = require('gulp-jshint'),
-    stylish = require('jshint-stylish'),
-    rev = require('gulp-rev'),
-    uglify = require('gulp-uglify'),
-    del = require('del');
-    usemin = require('gulp-usemin')
+var gulp = require('gulp');
+var useref = require('gulp-useref');
```

Comment:
I think there's some debate on which one is better, but we can revisit this later once we pick a styleguide?

I was thinking of using `var $ = require('gulp-load-plugins');` but I thought we could do that later. I didn't want to change too much! I know I already have changed quite a bit haha